controllers/cloud.redhat.com/providers/web/local.go

```diff
@@ -486,24 +490,29 @@ func (web *localWebProvider) createIngress(app *crd.ClowdApp, deployment *crd.De
 				Host: web.Env.Status.Hostname,
 				IngressRuleValue: networking.IngressRuleValue{
 					HTTP: &networking.HTTPIngressRuleValue{
-						Paths: []networking.HTTPIngressPath{{
-							Path:     fmt.Sprintf("/api/%s/", apiPath),
-							PathType: (*networking.PathType)(utils.StringPtr("Prefix")),
-							Backend: networking.IngressBackend{
-								Service: &networking.IngressServiceBackend{
-									Name: nn.Name,
-									Port: networking.ServiceBackendPort{
-										Name: "auth",
-									},
-								},
-							},
-						}},
+						Paths: []networking.HTTPIngressPath{},
 					},
 				},
 			},
 		},
 	}
 
+	for _, apiPath := range apiPaths {
```

Comment:
I just updated it to use the same approach and format the strings first.

Re the second question -- ok, yes we can do that here if you want to. I think I should just be able to use a `+kubebuilder:validation:Pattern=<regex pattern>` annotation on the field, correct? What type of patterns do we want to permit here to start with? Only `/api/<text>/`? Technically I think the regex for this will be something like:

```
//+kubuilder:validation:Pattern="^\/api\/[a-zA-Z-]+\/$"
```